playbooks/compute/nto/roles/configurecluster/tasks/create_directory.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Check if artifacts folder exists
+  ansible.builtin.stat:
+    path: "{{ dir_path }}"
+  register: register_name
```

Comment:
Register it to a better name, like artifact_folder or something...

Reply:
This is a General use
it was use twice the name `artifacts` is not correct I will fix it